### PR TITLE
Add network policy support

### PR DESCRIPTION
**Pull request by jacoelho**
Implements: https://rancher.com/docs/rancher/v1.6/en/rancher-services/network-policy/

```terraform
resource "rancher_environment" "default" {
  name               = "default"
  orchestration  = "cattle"
  default_policy = "deny"

  policy = {
    action     = "allow"
    between = "com.rancher.foo"
  }
}
```

### CODE REVIEW

File: rancher/data_source_rancher_certificate.go [hunk 1]
```diff
@@ -76,8 +76,8 @@ func dataSourceRancherCertificateRead(d *schema.ResourceData, meta interface{})
 	name := d.Get("name").(string)
 	log.Printf("[INFO] Refreshing Rancher Certificate: %s", name)
 
-	environmentId := d.Get("environment_id").(string)
-	client, err := meta.(*Config).EnvironmentClient(environmentId)
+	environmentID := d.Get("environment_id").(string)
+	client, err := meta.(*Config).EnvironmentClient(environmentID)
```
Comment (raphink) [inline rancher/data_source_rancher_certificate.go]: This seems unrelated to this PR.

**Comment by raphink** [comment]
@jacoelho 	do you still want to merge this feature?

**Comment by jacoelho** [comment]
Moved away from rancher, sorry.

**Comment by raphink** [comment]
Thanks for the answer. I'll close this.
